test/functionality/dataflow/query/dependencies-query.test.ts

```diff
@@ -122,11 +123,18 @@ describe.sequential('Dependencies Query', withShell(shell => {
 	});
 
 	describe('Sourced files', () => {
+		for(const sourceFn of [
+			'source_url',
+			'source_gist'
+		] as const) {
+			testQuery(`${sourceFn}`, `${sourceFn}("a")` , { sourcedFiles: [{ nodeId: `1@${sourceFn}`, functionName: sourceFn, file: 'a' }] });
+		}
+
 		testQuery('Single source', 'source("test/file.R")', { sourcedFiles: [{ nodeId: '1@source', functionName: 'source', file: 'test/file.R' }] });
 
 		describe('Custom', () => {
 			const sourceCustomFile: Partial<DependenciesQuery> = {
-				sourceFunctions: [{ name: 'source.custom.file', argIdx: 1, argName: 'file' }]
+				sourceFunctions: [{ package: 'custom', name: 'source.custom.file', argIdx: 1, argName: 'file' }]
```

Comment:
I do not think we should act packages here, given that they are optional and this way we are unable to resolve the `custom` package in the future.
```suggestion
				sourceFunctions: [{ name: 'source.custom.file', argIdx: 1, argName: 'file' }]
```